modules/dedicated-configuring-your-application-routes.adoc

```diff
@@ -13,7 +13,7 @@ four-character string that is communicated to you after initial provisioning.
 
 If you want to use custom domain names for your application routes, you should
 set up a CNAME record in your DNS host to point to
-*_elb.<shard-id>.<cluster-id>.openshiftapps.com_*. While `elb` is recommended as a
+*_elb.apps.<cluster-id>.<shard-id>.openshiftapps.com_*. While `elb` is recommended as a
```

Comment:
There's more in the doc that needs to be updated. I'm working with SRE to get the right terminology, then I'll submit a PR against your branch for the updates.

Reply:
Sounds good!

Reply:
https://github.com/sheriff-rh/openshift-docs/pull/1

Reply:
Merged! Thanks